packages/common/src/search/types/IHubSearchOptions.ts

```diff
@@ -47,6 +47,11 @@ export interface IHubSearchOptions {
    * DEPRECATED: Use `include` instead
    */
   enrichments?: Enrichments[];
+
+  /**
+   * Specify what entity to search for; For use with hubSearch
+   */
+  targetEntity?: EntityType;
```

Comment:
why is this part of options and query?

Reply:
If you pass in a string to `search(...)` we need a means to specify what entity we're searching against. It is duplicative if  you pass in an `IQuery` which is why it's optional